### PR TITLE
VATRP-2941: Fix no video on logout

### PR DESCRIPTION
**Pull request by zackm0571**

### CODE REVIEW

File: Classes/WizardViewController.m [hunk 1]
```diff
@@ -1703,11 +1703,6 @@ - (void)enableAppropriateCodecs:(const MSList *)codecs {
         pt = (PayloadType *)elem->data;
         linphone_core_enable_payload_type(lc, pt, [[DefaultSettingsManager sharedInstance].enabledCodecs containsObject:[NSString stringWithUTF8String:pt->mime_type]]);
         
-        // download_bandwidth , related to the document
```
Comment (shareefalis) [inline Classes/WizardViewController.m:1706]: No longer needed?

Comment (shareefalis) [inline Classes/WizardViewController.m:1706]: I am pretty sure that was the cause of the problem.

File: Classes/SettingsViewController.m [hunk 2]
```diff
@@ -1067,12 +1067,8 @@ - (void)settingsViewController:(IASKAppSettingsViewController *)sender
         [alert addCancelButtonWithTitle:NSLocalizedString(@"Cancel", nil) block:nil];
         [alert addButtonWithTitle:NSLocalizedString(@"Launch Wizard", nil)
                             block:^{
-                                NSString *appDomain = [[NSBundle mainBundle] bundleIdentifier];
```
Comment (shareefalis) [inline Classes/SettingsViewController.m:1070]: try the change  with this reverted

**Comment by ianblenke** [comment]
+1 Go for it.

**Pull request merged**